Docs 6415 v3 duke (#1119)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/about-mule-configuration.adoc

> endif::[]

Reviewer: HERE

:keywords: configuration, deploy, mule

> All Mule applications, domain, and policies are configured via an XML DSL.

Author: This is consolidating content from a section that was removed from the mule-application-about.adoc, which itself has been renamed mule-components.adoc

Reviewer: Does "All Mule" apply to all three of "applications," "domain," and "policies"? Reword to clarify.
Note also that "all applications, domain, and policies" violates number agreement ("all domain").

Reviewer: Also, should we reword for active voice: "You configure"?

Author: Generally, we want people configuring them through the UI. We're trying to say that underlying that is an XML.

This XML file specifies the resources that compose the artifact, including
dependencies needed to run the Mule application.

> Though it is possible to write the XML by hand, it is more common to

Reviewer: though > although
GLOBAL in all writing.

Reviewer: "by hand" > "manually"

Reviewer: ```suggestion
Although you can write the XML file manually, it is more common to
```

> use the graphical user interfaces in Design Center or Anypoint Studio to structure and define the behavior of your Mule app. While you use either of these GUIs, the underlying XML is provided for you. 

Reviewer: ```suggestion
use the graphical user interfaces in Anypoint Design Center or Anypoint Studio to structure and define the behavior of your Mule app. While you use either of these GUIs, the underlying XML is provided for you. 
```
